pkg/alertmanager/alerts/store.go

```diff
@@ -0,0 +1,64 @@
+package alerts
+
+import (
+	"context"
+
+	"github.com/cortexproject/cortex/pkg/configs"
+	"github.com/cortexproject/cortex/pkg/configs/client"
+)
+
+// AlertStore stores and configures users rule configs
+type AlertStore interface {
+	ListAlertConfigs(ctx context.Context) (map[string]AlertConfigDesc, error)
+}
+
+// ConfigAlertStore is a concrete implementation of RuleStore that sources rules from the config service
+type ConfigAlertStore struct {
```

Comment:
`ConfigDBAlertStore` would be more appropriate? _Just asking, up to you._

Reply:
SGTM